docs/docs/configuration/docker_vars/index.html

```diff
@@ -584,7 +590,7 @@ <h2 id="direct-mapping-to-reaper-specific-configuration-settings">Direct Mapping
 
 <tr>
 <td><code>REAPER_LOGGING_FORMAT</code></td>
-<td><code>REAPER_LOGGING_APPENDERS_CONSOLE_LOG_FORMAT</code></td>
+<td><code>REAPER_LOGGING_APPENDERS_LOG_FORMAT</code></td>
```

Comment:
This is being changed incorrectly. I may have missed this fix when I was updating the documentation.

It should be: `REAPER_LOGGING_APPENDERS_CONSOLE_LOG_FORMAT`.

Sorry about that. Should I fix this before we merge this PR?